README.md

```diff
@@ -125,34 +125,35 @@ test('Fetch makes an API call and displays the greeting when load-greeting is cl
 ## Table of Contents
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
+
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Installation](#installation)
```

Comment:
I'm guessing this has something to do with you having an old version of prettier. Could you make sure that you have the latest version?

Reply:
Sure, I'll get this done in the next few hours.